website/docs/d/event_orchestration_integration.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: 'pagerduty'
+page_title: 'PagerDuty: pagerduty_event_orchestration_integration'
+sidebar_current: 'docs-pagerduty-datasource-event-orchestration-integration'
+description: |-
+  Get information about an Integration for an Event Orchestration that you have created.
+---
+
+# pagerduty_event_orchestration_integration
+
+Use this data source to get information about a specific [Integration][1] for an Event Orchestration.
+
+## Example Usage
+
+```hcl
+
+data "pagerduty_event_orchestration_integration" "integration" {
+  event_orchestration = "19acac92-027a-4ea0-b06c-bbf516519601"
+  id = "1b49abe7-26db-4439-a715-c6d883acfb3e"
+  label = "Example integration"
+}
+
+```
```

Comment:
Can we include an example where we reference a Terraformed orchestration? Something like this:
```hcl
resource "pagerduty_event_orchestration" "orch" {
  name = "..."
}

data "pagerduty_event_orchestration_integration" "integration" {
  event_orchestration = pagerduty_event_orchestration.orch.id
  label = "Example integration"
}
```